src/Epsagon.Dotnet.Lambda/EpsagonHandler.cs

```diff
@@ -244,5 +249,11 @@ private static void HandleInstrumentationError(Exception ex)
 
             InstumentationExceptionsCollector.Exceptions.Add(ex);
         }
+        private static void setTraceUrl(string partialTraceUrl)
```

Comment:
functions are PascalCase in c# please update all relevant funcs

Reply:
fixed

Reply:
fixed